docs/working-with-spacy.md

```diff
@@ -18,21 +18,21 @@ There are many other [models](https://spacy.io/usage/models) supporting differen
 
 ## Search
 
-Here's a sample pre-built index on TREC Washington Post to play with (used in the TREC 2018 Core Track):
+Build index on TREC Washington Post Corpus (used in TREC 2018 Core Track) with `-storeContents` option:
 
 ```bash
-# TODO: upload pre-built index storing contents
-wget index-core18-contents.tar.gz
-tar xvfz index-core18-contents.tar.gz -C indexes
-rm index-core18-contents.tar.gz
+sh target/appassembler/bin/IndexCollection -collection WashingtonPostCollection -generator WashingtonPostGenerator \
+  -input /path/to/WashingtonPost -index index-core18-contents -threads 8 -storePositions -storeDocvectors -storeContents
```

Comment:
put index in `indexes/` directory?

Also, can use `python -m pyserini.index` - see https://github.com/castorini/pyserini/blob/master/docs/experiments-msmarco-passage.md